### ISSUE TITLE
HTML tags not working in WP4.7

### ISSUE THREAD

**Issue by warrenoshita** [smooch/smooch-wordpress#18]
Hi, I just created my first bot and deployed it on my Wordpress site running v4.7. Tested content tagged with italics and href in the IDE and they format properly there, but don't work on my site. They look like raw HTML.

Tested in FF 50.1.0 and Chrome 55.0.2883.95 (64-bit) both on macOS 10.11.6 (15G1108).
![screen shot 2017-01-09 at 4 31 16 pm](https://cloud.githubusercontent.com/assets/24486844/21789344/1a493574-d689-11e6-888b-08331d9d7e01.png)

**Comment by jpjoyal** [comment]
Smooch does not yet support formatting in messages. I'll note your interest for this feature. @warrenoshita are  you looking into doing more than `<i>`,`<a>` &`<b>` ?

**Comment by warrenoshita** [comment]
No, simple text formatting and links would work well. I ended up going straight from motion.ai to my website and the formatting worked perfectly. But if my use case involved more channels, I would want to use smooch.

**Comment by warrenoshita** [comment]
To further clarify, I also bypassed WP and published on a regular HTML page created just for the bot.
